packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -305,6 +320,8 @@ export class ColorPicker
 
   private internalColorUpdateContext: "internal" | "initial" | "user-interaction" | null = null;
 
+  private isClearable: boolean = !!(this.clearable || this.allowEmpty);
```

Comment:
Per our previous conversation, `isClearable` should be assigned in a lifecycle hook.